soc/nordic/soc.yml

```diff
@@ -68,6 +68,7 @@ runners:
       - runners:
```

Comment:
is --recover not supported for nrf54h20?

Reply:
No, nrfutil currently does not support this operation.